rtlsdr/librtlsdr.py

```diff
@@ -58,76 +58,97 @@ def load_librtlsdr():
 
 # int rtlsdr_get_device_usb_strings(uint32_t index, char *manufact,
 #                                   char *product, char *serial)
+# * \return 0 on success
 f = librtlsdr.rtlsdr_get_device_usb_strings
 f.restype, f.argtypes = c_int, [c_uint,
                                 POINTER(c_ubyte),
                                 POINTER(c_ubyte),
                                 POINTER(c_ubyte)]
 
 # int rtlsdr_open(rtlsdr_dev_t **dev, uint32_t index);
+# FIXME: return value not documented
```

Comment:
Although there is no indication of what the error response should be, we do know that -1 indicates an error from experience. I believe we should assume that a return of <0 should be treated as an error, but raising an issue with librtlsdr would be a good way to avoid "assuming" things